.kokoro/tests/run_test_java.sh

```diff
@@ -60,6 +60,23 @@ if [[ "$file" == *"functions/helloworld/"* ]]; then
     fi
 fi
 
+# Build and deploy Appengine flex samples
+if [[ "$file" == *"flexible/java-11/"* ]]; then
```

Comment:
Can we deploy everything under `flexible/`?

Reply:
+1 (and to the similar comment [here](https://github.com/GoogleCloudPlatform/java-docs-samples/pull/8165#discussion_r1203155868)).

If there's a reason we have to limit the scope, these filters should be a constant at the top of each file. Also a comment about why we selected these specific folders to be tested should be included.